sphinx/builders/html.py

```diff
@@ -283,10 +320,54 @@ def create_build_info(self):
         # type: () -> BuildInfo
         return BuildInfo(self.config, self.tags, ['html'])
 
+    def init_css_files(self):
+        # type: () -> None
+        for filename, alternate, title in self.app.registry.stylesheets:
+            self.add_stylesheet(filename, alternate, title)
+
+        for stylesheet in self.get_builder_config('stylesheets', 'html'):
+            if isinstance(stylesheet, string_types):
+                self.add_stylesheet(stylesheet)
+            else:
+                try:
+                    filename, alternate, title = stylesheet
+                    self.add_stylesheet(filename, alternate, title)
+                except (TypeError, ValueError):
+                    logger.warning('invalid stylesheet: %r', stylesheet)
+
+    def add_stylesheet(self, filename, alternate=False, title=None):
+        # type: (unicode, bool, unicode) -> None
+        if '://' not in filename:
+            # make a new path; the css file will be copied to ``_static`` directory
+            filename = posixpath.join('_static', filename)
+        if alternate:
+            rel = u'alternate stylesheet'
+        else:
+            rel = u'stylesheet'
+        css = Stylesheet(filename, title, rel)  # type: ignore
+        self.css_files.append(css)
+
+    def init_script_files(self):
+        # type: () -> None
+        for filename in self.app.registry.javascripts:
+            self.add_javascript(filename)
+
+        for filename in self.get_builder_config('javascripts', 'html'):
+            self.add_javascript(filename)
+
+        if self.config.language and self._get_translations_js():
+            self.add_javascript('translations.js')
+
+    def add_javascript(self, filename):
+        # type: (unicode) -> None
+        if '://' not in filename:
+            filename = posixpath.join('_static', filename)
```

Comment:
I think `_static` is a configurable path?

Reply:
At present, it is not configurable.